tests/api/test_incidents.py

```diff
@@ -0,0 +1,138 @@
+from django.urls import reverse
+from faker import Faker
+from rest_framework.test import force_authenticate
+import json
+import pytest
+import random
+
+from response.models import Incident, ExternalUser
+from response import serializers
+from response.core.views import IncidentViewSet
+from tests.factories import IncidentFactory
+
+faker = Faker()
+
+
+def test_list_incidents(arf, api_user):
+    persisted_incidents = IncidentFactory.create_batch(5)
+
+    req = arf.get(reverse("incident-list"))
+    force_authenticate(req, user=api_user)
+    response = IncidentViewSet.as_view({"get": "list"})(req)
+
+    assert response.status_code == 200, "Got non-200 response from API"
+    content = json.loads(response.rendered_content)
+
+    assert "results" in content, "Response didn't have results key"
+    incidents = content["results"]
+    assert len(incidents) == len(
+        persisted_incidents
+    ), "Didn't get expected number of incidents back"
+
+    for idx, incident in enumerate(incidents):
+        assert incident["report_time"]
+
+        # incidents should be in order of newest to oldest
+        if idx != len(incidents) - 1:
```

Comment:
You could store `report_time` outside the loop and update/compare it on each iteration to simplify here, not a blocker though.